src/show/pages/private/Admin/index.jsx

```diff
@@ -13,68 +13,40 @@ import {
   addConstructor,
   getConstructors
 } from 'process/slices/constructorsSlice';
+import { updateProps } from 'process/slices/transientSlice';
 import { addEvent, getEvents } from 'process/slices/eventsSlice';
 import { addDriver, getDrivers } from 'process/slices/driversSlice';
 
 const Admin = () => {
-  const [createConstructorModal, setCreateConstructorModal] = useState(false);
-  const [createDriverModal, setCreateDriverModal] = useState(false);
-  const [createEventModal, setCreateEventModal] = useState(false);
   const [search, setSearch] = useState('');
```

Comment:
Also put the searchQuery on the transient state. And when this component mounts, you will have to initialize the transient state, the `searchQuery` (or however you wanna call it) key with `''` (otherwise it will be undefined and we do not want that).